package br.unisc.numerosMagicos.excecoes;

public class ResultadoEncontradoException extends Exception {

}